extensions/packageManager/src/node/index.ts

```diff
@@ -247,30 +132,30 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
     },
     getFeed: async function (req, res) {
       // why an array? In the future it is feasible we would want to mix several feeds together...
-      const packageSources = [req.query.url];
+      const packageSources: IPackageSource[] = [req.query];
 
-      let combined = [];
-      for (const url of packageSources) {
+      let combined: IPackageDefinition[] = [];
+      for (const packageSource of packageSources) {
         try {
-          let raw;
-          if (fs.existsSync(url)) {
-            const rawlocal = await crawlLocalFeed(url);
-            // cast this to the form of the http response from nuget
-            raw = { data: { data: rawlocal } };
-          } else {
-            raw = await axios.get(url);
-          }
-          const feed = await normalizeFeed(raw.data);
-          if (Array.isArray(feed)) {
-            combined = combined.concat(feed);
+          const feed: IFeed = await new FeedFactory(composer).build(packageSource);
+          const packageQuery: IPackageQuery = {
+            prerelease: true,
+            semVerLevel: '2.0.0',
+            query: 'tags:bf-component',
+          };
+
+          const packages = await feed.getPackages(packageQuery);
+
+          if (Array.isArray(packages)) {
+            combined = combined.concat(packages);
```

Comment:
I think `combined.push(...packages)` is the more idiomatic way to do this.

Reply:
This is the original code that just moved and I'm trying to reduce risk. So, I'll change it as part of this PR if I get unit tests in. Otherwise I'll make this change along with unit tests to be sure I exercise this path, since in the current UI this is always a single element array  in practical terms.

Reply:
Added this one after all, thanks for the suggestion!